.github/workflows/ci-integration-tests.yml

```diff
@@ -0,0 +1,164 @@
+name: "CI - Integration Tests"
+
+on:
+  push:
+    paths:
+      # NOTE: GitHub Actions do not allow using YAML references, the same path
+      # list is used below for the pull request event. Keep both lists in sync!!
+
+      # this file itself
+      - .github/workflows/ci-integration-tests.yml
+
+      # the web frontend
+      - web/**.json
```

Comment:
I guess these globs are recursive, right?

Reply:
"*" matches anything except "/", "**" matches really anything, including "/"
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet